UNBOUND_NAME[2000] in test/integ/function_def_scoping.error.pytch, line 2, character 1:
Error: I couldn't find a binding in the current scope with the name 'x'.
   +-----------------------------------------------------------------------+
   | test/integ/function_def_scoping.error.pytch                           |
 1 | def foo(x) => x                                                       |
 2 | x                                                                     |
   | ^ Error: I couldn't find a binding in the current scope with the name |
   | 'x'.                                                                  |
   +-----------------------------------------------------------------------+
